page-data.php

```diff
@@ -50,7 +50,8 @@
       '#js-TJIChartView',  // Element to build the charts into
       '#secondary',  // Element to put the filter panel inside
       '<div class="col-sm-12 col-lg-6" />',  // Wrapper for chart canvases
-      '<div class="col-sm-12">{count} records</div>'  // Template for record count element
+      '<div class="col-sm-12 count">{count} records total</div>',  // Template for record count element
+      '<div class="loader"></div>'  // HTML for loading element, displayed while data is fetching
```

Comment:
Instead of adding to the long list of parameters, maybe this div could just live in the markup and be overwritten by the chart content. or be a class on the container div, that chartview is responsible for removing.

Reply:
donezo